### PR TITLE
protontricks: refresh page

### PR DESCRIPTION
**Pull request by kbdharun**
- [x] The page(s) are in the correct platform directories: `linux`.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

## Changes

- Updating page description, and help command.
-  Adding new commands.

### CODE REVIEW

File: pages/linux/protontricks.md [hunk 1]
```diff
@@ -18,3 +18,11 @@
 - Run a command within a game's installation directory:
 
 `protontricks -c {{command}} {{appid}}`
+
+- List all installed games:
+
+`protontricks -l`
+
+- Search for a game's App ID by entering it's name:
```
Comment (adrienthebo) [inline pages/linux/protontricks.md:26]: **fixup:** "it's" is used as a contraction of "it is", "its" is the possessive form. We can also avoid this entirely:

```suggestion
- Search for a game's App ID by name:
```

**Review by adrienthebo — APPROVED** [review]
LGTM!

File: pages/linux/protontricks.md [hunk 2]
```diff
@@ -1,11 +1,11 @@
 # protontricks
 
-> A simple wrapper that does Winetricks things for Proton enabled games, requires Winetricks.
+> A simple wrapper that runs Winetricks commands for Proton enabled games.
 > More information: <https://github.com/Matoking/protontricks>.
 
 - Show the protontricks help message:
 
-`protontricks`
+`protontricks --help`
```
Comment (EmilySeville7cfg) [inline pages/linux/protontricks.md:8]: Put help code examples to the end.

File: pages/linux/protontricks.md [hunk 3]
```diff
@@ -18,3 +18,11 @@
 - Run a command within a game's installation directory:
 
 `protontricks -c {{command}} {{appid}}`
+
+- List all installed games:
```
Comment (EmilySeville7cfg) [inline pages/linux/protontricks.md:22]:
```suggestion
- [l]ist all installed games:
```

You can consider option case as more important thing than letter case in code description. That's why you can change `L` to `l` and not use `[L]`.

File: pages/linux/protontricks.md [hunk 4]
```diff
@@ -18,3 +18,11 @@
...
+
+`protontricks -l`
+
+- Search for a game's App ID by name:
```
Comment (EmilySeville7cfg) [inline pages/linux/protontricks.md:26]:
```suggestion
- [s]earch for a game's App ID by name:
```

Please use mnemonics as much as possible. If you don't know which mnemonic to use we can help. ;)

File: pages/linux/protontricks.md [hunk 5]
```diff
@@ -18,3 +18,11 @@
...
+
+`protontricks -s {{Game_Name}}`
```
Comment (EmilySeville7cfg) [inline pages/linux/protontricks.md:28]:
```suggestion
`protontricks -s {{game_name}}`
```

Just snake_case, nothing more is permitted unless it's a placeholder sample value.

**Review by EmilySeville7cfg — CHANGES_REQUESTED** [review]

File: pages/linux/protontricks.md (hunk 4 above)
Comment (kbdharun) [reply pages/linux/protontricks.md:26]: > Please use mnemonics as much as possible. If you don't know which mnemonic to use we can help. ;)

Sure, I will try to use more mnemonics in future, I was just confused about whether to use them in commands with single arguments.

**Review by EmilySeville7cfg — APPROVED** [review]

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**